### PR TITLE
Nymphotrait

### PR DESCRIPTION
**Pull request by NopemanMcHalt**
[Changelogs]: # (Your PR should contain a detailed changelog of notable changes, titled and categorized appropriately. This includes, new features, sprites, sounds, balance changes, admin tools, map edits, removals, big refactors, config changes, hosting changes and important fixes. An example changelog has been provided below for you to edit. If you need additional help, read https://github.com/tgstation/tgstation/wiki/Changelogs)

:cl: Anonymous
add: Adds nymphomania trait, which will raise your minimal arousal and boost rate of it.
/:cl:

[why]: # 
I did this out of boredom. **Not sure about values, though.**

### CODE REVIEW

**Review by deathride58 — CHANGES_REQUESTED** [review]
Move to modular_citadel/code/datums/traits/neutral.dm

File: code/datums/traits/neutral.dm [hunk 1]
```diff
@@ -31,3 +31,30 @@
 	var/datum/species/species = H.dna.species
 	species.liked_food = initial(species.liked_food)
 	species.disliked_food = initial(species.disliked_food)
+
+//Citadel Additions starts there
+
+/datum/trait/libido
+	name = "Nymphomania"
+	desc = "You're always feeling a bit in heat. You're also get in steamy mood slightly faster than usual."
```
Comment (mattmatterson111) [inline code/datums/traits/neutral.dm]: "You're also get in steamy mood"
Bad grammar.

**Comment by NopemanMcHalt** [comment]
@deathride58, consider it done.

**Review by deathride58 — APPROVED** [review]
Yiff! ✊

**Pull request merged**